client/css/style.css

```diff
@@ -752,6 +752,71 @@ button,
 	color: #50a656 !important;
 }
 
+#chat .user.color-1 { color: #1396cf; }
```

Comment:
Your indexes are off. Either you need to add `1` in `colorClass` function or re-index classes to be 0-63.